scripts/wi-setup.sh

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_PATH="$(dirname "${BASH_SOURCE[0]}")"
+
+validate() {
+    # check if user is logged into azure cli
+    if ! az account show > /dev/null 2>&1; then
+        echo -e "\e[31mPlease login to Azure CLI using 'az login'\e[0m"
+        exit 1
+    fi
+}
+
+create_azure_blob_storage_account() {
+    export RESOURCE_GROUP="oidc-issuer"
+    export LOCATION="westus2"
+    if [ $(az group exists --name "${RESOURCE_GROUP}") == 'false' ]; then
```

Comment:
```suggestion
    if [ $(az group exists --name "${RESOURCE_GROUP}" --output tsv) == 'false' ]; then
```

It's safer to specify the output type. My `az` config defaults to JSON, so this command would fail for me:

```shell
% az group exists -n foo             
Result
--------
False
```